test/view/iota.cpp

```diff
@@ -34,20 +35,40 @@ struct Int
 
 template<typename Integral>
 void test_iota_minus() {
-  using namespace ranges;
-  using D = detail::iota_difference_t<Integral>;
-  using I = Integral;
-  Integral max = std::numeric_limits<Integral>::max();
-
-  CHECK(detail::iota_minus_(I(0), I(0)) == D(0));
-  CHECK(detail::iota_minus_(I(0), I(1)) == D(-1));
-  CHECK(detail::iota_minus_(I(1), I(0)) ==  D(1));
-  CHECK(detail::iota_minus_(I(1), I(1)) == D(0));
-
-  CHECK(detail::iota_minus_(I(max - I(1)), I(max - I(1))) == D(0));
-  CHECK(detail::iota_minus_(I(max - I(1)), I(max)) == D(-1));
-  CHECK(detail::iota_minus_(I(max), I(max - I(1))) == D(1));
-  CHECK(detail::iota_minus_(I(max), I(max)) == D(0));
+    using namespace ranges;
+    using D = detail::iota_difference_t<Integral>;
+    using I = Integral;
+    Integral max = std::numeric_limits<Integral>::max();
+
+    CHECK(detail::iota_minus_(I(0), I(0)) == D(0));
+    CHECK(detail::iota_minus_(I(0), I(1)) == D(-1));
+    CHECK(detail::iota_minus_(I(1), I(0)) ==  D(1));
+    CHECK(detail::iota_minus_(I(1), I(1)) == D(0));
+
+    CHECK(detail::iota_minus_(I(max - I(1)), I(max - I(1))) == D(0));
+    CHECK(detail::iota_minus_(I(max - I(1)), I(max)) == D(-1));
+    CHECK(detail::iota_minus_(I(max), I(max - I(1))) == D(1));
+    CHECK(detail::iota_minus_(I(max), I(max)) == D(0));
+}
+
+template <typename U>
+void test_wrap_around() {
+    using namespace ranges;
+    auto max = std::numeric_limits<U>::max();
```

Comment:
The fact that `[U(-3), U(2))` is a valid range of exactly 5 elements whether or not `U` is a signed or unsigned integral type is the primary motivation for not requiring `to < from` for ranges of unsigned types.